### PR TITLE
Enable httpclient 5 test

### PR DESCRIPTION
**Pull request by vuqtran88**
- Now .NET 5 is GA, the `HttpClientInstrumentationNet5`  can now be enabled to run as part of GHA.
- Updates the Changelog to announce .NET 5 GA support.

### CODE REVIEW

File: src/Agent/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Fixes Issue [#XX](https://github.com/newrelic/newrelic-dotnet-agent/issues/XX)
 
 ## [8.35] - 2020-11-09
 
+### New Features
+* **.NET 5 GA Support** <br/>
+We have validated that this version of the agent is compatible with .NET 5 GA. See the [compatibility and requirements for .NET Core](https://docs.newrelic.com/docs/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core) page for more details.
```
Comment (angelatan2) [inline src/Agent/CHANGELOG.md:17]: Vu, 
Could you add,  
Moving forward, information on supported .NET versions will be reserved for the compatibility matrix alone.

File: .github/workflows/all_solutions.yml [hunk 2]
```diff
@@ -443,6 +443,21 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Setup .NET Core 2.2.103
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '2.2.103'
+
+      - name: Setup .NET Core 3.1.100
```
Comment (jaffinito) [inline .github/workflows/all_solutions.yml:451]: 3.1.100 is already setup:
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#net-core-sdk

**Review by jaffinito — APPROVED** [review]
After discussing this, its looks fine to me now.

File: src/Agent/CHANGELOG.md (hunk 1 above)
Comment (vuqtran88) [reply src/Agent/CHANGELOG.md:17]: I like to keep it as it is for now. I know we already discussed this with the team about not mentioning it in the release note, but I now can see the benefit of having this information in the release note.

**Pull request merged**